VPLAY-11251 AAMP-JS ( jsbinding ) prints are not genereated

Abhi-jith-S opened the pull request:
Reason for change : Enabling USE_ETHAN_LOG macro before building aampjsbinding

Test Procedure: See ticket

Risks: Low

shripadgithub committed:
VPLAY-11251  : [VIPA] AAMP-JS ( jsbinging ) prints are not genereated

Reason for change: adding debug logs

Test Procedure:

Priority: P2

Risks:Low

shripadgithub committed:
VPLAY-11251  : debug log in aamp

Reason for change:

Test Procedure:

Priority: P2

Risks:Low

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:
Test Procedure:
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:additional logs added(ethan)
Test Procedure:
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:added to confirm usegae of ethanlog
Test Procedure:
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:ethan logging specific
Test Procedure:
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
Revert "VPLAY-11251  : debug log in aamp"

This reverts commit 9f78736448bf27dc41e00f1135729cf2c5a88e5d.

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:attribute const. method of printing with filename
Test Procedure:NONE
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : debug log in aamp

Reason for change:CMakeFiles changed
Test Procedure:NONE
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : Fix for js logging

Reason for change:To enable Ethan logging macro
Test Procedure:NONE
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : Fix for js logging

Reason for change:Fix added for js logging
Test Procedure:Refer ticket
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : Fix for js logging

Reason for change:Removed redudant log lines
Test Procedure:Refer ticket
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : Fix for js logging

Reason for change:Removing redundant debug logs
Test Procedure:refer ticket
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=597) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=597) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: shripadgithub<br/>:x: asasik397<br/><hr/>**asasik397** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=597) it.</sub>

Vinish100 commented on CMakeLists.txt:
```diff
@@ -213,7 +213,18 @@ install(TARGETS tsb
 		PUBLIC_HEADER DESTINATION include
 )
+#Enabling  ethan logging macro
+if(CMAKE_USE_ETHAN_LOG)
```
Please also move the CMAKE_SYSTEMD_JOURNAL block to here so that the LIBAAMPJSBINDINGS_DEFINES macro is updated properly

Commit:
VPLAY-11251  : Fix for js logs

Reason for change: Included systemd logging as well.
Test Procedure:Refer ticket
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

Commit:
VPLAY-11251  : Fix for js logging

Reason for change:Minor corrections
Test Procedure:Refer text
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>
